code/modules/antagonists/nations/factions/science.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationsci
+	name = "Sciencetopia"
```

Comment:
You could probably come up with a better name for this

Reply:
Sciencia gang come on